### PR TITLE
Small fixes

### PR DESCRIPTION
**Pull request by TingALin**
Small fixes , reverted original PR purpose.

### CODE REVIEW

**Comment by rossbulat** [comment]
Is this ready to be reviewed?

**Comment by TingALin** [comment]
> Is this ready to be reviewed?

yes

**Review by rossbulat — CHANGES_REQUESTED** [review]

File: src/modals/UnlockChunks/Overview.tsx [hunk 1]
```diff
@@ -7,26 +7,29 @@ import { ButtonSubmit } from '@rossbulat/polkadot-dashboard-ui';
...
+import { forwardRef, useEffect } from 'react';
 import { useTranslation } from 'react-i18next';
 import { planckToUnit } from 'Utils';
 import { NotesWrapper, Separator } from '../Wrappers';
 import { ChunkWrapper, ContentWrapper } from './Wrappers';
 
 export const Overview = forwardRef(
   ({ unlocking, bondFor, setSection, setUnlock, setTask }: any, ref: any) => {
-    const { network, consts } = useApi();
+    const { network, consts, status: connectionStatus } = useApi();
```
Comment (rossbulat) [inline src/modals/UnlockChunks/Overview.tsx:24]:
```suggestion
    const { network, consts, apiStatus } = useApi();
```

`status` almost always requires an alias for this. Can you rename `status` to `apiStatus` and also update `connectionStatus` in `contexts/API` to `apiStatus` also.

Comment (TingALin) [reply src/modals/UnlockChunks/Overview.tsx]: Done, just leave  [type ConnectionStatus](https://github.com/TingALin/polkadot-staking-dashboard/blob/ting-unlock-era/src/contexts/Api/types.ts#L9 )  unchanged as i think this's more intuitive

Comment (rossbulat) [reply src/modals/UnlockChunks/Overview.tsx]: Why change it for this one file?

**Pull request merged**